### PR TITLE
Enforce a specific yarn version

### PR DESCRIPTION
**Pull request by qgolsteyn**
### Description
An issue I believe we will soon run into is inconsistent version of yarn (the dependency manager for the visualizer) between the CI and the various developers. Some of the PRs already being pushed right now are showing signs of inconsistent yarn versions (see #240), most visible thanks to the high diff of `yarn.lock` despite the fact that this PR introduces a small set of new dependencies.

One way to prevent inconsistent versions of yarn across machines is by committing yarn into the repository. Well... I know this is not the _greatest_ way of dealing with this problem. It should not come as a surprise however that this is [what the developers of yarn suggest doing](https://yarnpkg.com/lang/en/docs/yarnrc/#toc-yarn-path) (Yay Javascript).

I am open to suggestions, but I recommend locking yarn moving forward with this project. Again, this will ensure that we avoid any "this works on my machine" errors.

Changes done:
- Commit yarn into repo (sigh)
- Enforce the yarn version we committed (so you cannot run the project without the right version)
- Tell yarn to use the version we have committed in the project

### Testing Done
Manual testing

### Review Checklist
*(Please check every item to indicate your code complies with it (by changing `[ ]`->`[x]`). This will hopefully save both you and the reviewer(s) a lot of time!)*
***It is the reviewers responsibility to also make sure every item here has been covered***
- [X] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
- [X] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom` 
- [X] **Remove all commented out code**
- [X] **Remove extra print statements**: for example, those just used for testing
- [X] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [X] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/corner_kick/.yarnrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Enforce a specific yarn version
+yarn-path "./scripts/yarn-1.13.0.js"
```
Comment (MathewMacDougall) [inline src/corner_kick/.yarnrc]: GIthub complaining about lack of newline

**Review by MathewMacDougall — CHANGES_REQUESTED** [review]
Unfortunate we have to do this, but I can't think of any other alternatives. We already do something similar with `clang-format`. Just make sure this is documented.

File: src/corner_kick/.yarnrc (hunk 1 above)
Comment (qgolsteyn) [reply src/corner_kick/.yarnrc]: Fixed

**Comment by qgolsteyn** [comment]
@MathewMacDougall I added a section in the README

**Review by henryymliu — APPROVED** [review]

**Review by MathewMacDougall — APPROVED** [review]

**Pull request merged**